[Docs] Upgrade list

okdas opened the pull request:
## Summary

Add recent Alpha TestNet upgrades

## Type of change

Select one or more from the following:

- [ ] New feature, functionality or library
- [ ] Consensus breaking; add the `consensus-breaking` label if so. See #791 for details
- [ ] Bug fix
- [ ] Code health or cleanup
- [x] Documentation
- [ ] Other (specify)

## Testing

- [x] **Documentation**: `make docusaurus_start`; only needed if you make doc changes
- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.

## Sanity Checklist

- [x] I have tested my changes using the available tooling
- [ ] I have commented my code
- [x] I have performed a self-review of my own code; both comments & source code
- [ ] I create and reference any new tickets, if applicable
- [x] I have left TODOs throughout the codebase, if applicable

okdas commented on tools/scripts/tx_jsons/staking_params.json:
```diff
@@ -0,0 +1,18 @@
+{
```
Just wanted to add these into the repo. I'll make use of them in the validator staking documentation.

Olshansk commented on docusaurus/docs/operate/quickstart/docker_compose_debian_cheatsheet.md:
```diff
@@ -110,6 +110,13 @@ source ~/.bashrc
 ## Start up the full node
 
+
+:::warn
```
```suggestion
:::warning
```

Olshansk commented on docusaurus/docs/operate/quickstart/docker_compose_walkthrough.md:
```diff
@@ -156,6 +156,11 @@ sed -i -e s/NODE_HOSTNAME=/NODE_HOSTNAME=69.42.690.420/g .env
 
 ### Launch the Node <!-- omit in toc -->
 
+:::warn
```
```suggestion
:::warning
```

Olshansk commented on docusaurus/docs/protocol/upgrades/upgrade_list.md:
```diff
@@ -18,21 +20,26 @@ The tables below provide a list of past and upcoming protocol upgrades. For more
+| [`v0.0.9-4`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-4) |    ❌    |    ✅     |                    ⚠️ [follow manual upgrade instructions](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-4) ⚠️                    | `46329`                                                                                                                      |
+| [`v0.0.9-3`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) |    ❌    |    ✅     | ❌ Active Alpha TestNet Participants Only: [follow manual upgrade instructions](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) | `17102`                                                                                                                      |
+| [`v0.0.9`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9)     |   N/A   |   N/A    |                                                               ❌ genesis version                                                                | N/A                                                                                                                          |
+### Syncing from genesis - manual steps
```
```suggestion

### Syncing from genesis - manual steps
```

Olshansk commented on docusaurus/docs/protocol/upgrades/upgrade_list.md:
```diff
@@ -18,21 +20,26 @@ The tables below provide a list of past and upcoming protocol upgrades. For more
+| [`v0.0.9-3`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) |    ❌    |    ✅     | ❌ Active Alpha TestNet Participants Only: [follow manual upgrade instructions](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9-3) | `17102`                                                                                                                      |
+| [`v0.0.9`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.9)     |   N/A   |   N/A    |                                                               ❌ genesis version                                                                | N/A                                                                                                                          |
+### Syncing from genesis - manual steps
+<!-- TODO(@okdas): when the next cosmovisor version released with `https://github.com/cosmos/cosmos-sdk/pull/21790` included - provide automated solution (csv file + pre-downloaded binaries) that will add hot-fixes automatically, allowing to sync from block #1 without any intervention -->
```
FYI that I believe this is live now: https://github.com/cosmos/cosmos-sdk/releases/tag/cosmovisor%2Fv1.7.0

okdas replied:
this is wonderful

bryanchriswhite commented on docusaurus/docs/protocol/upgrades/upgrade_list.md:
```diff
@@ -18,21 +20,26 @@ The tables below provide a list of past and upcoming protocol upgrades. For more
-| [`v0.0.4`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.4) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
+| Version                                                                      | Planned | Breaking |                                                          Requires Manual Intervention                                                          | Upgrade Height                                                                                                               |
+| ---------------------------------------------------------------------------- | :-----: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------------------- |
+| [`v0.0.10`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.10)   |    ✅    |    ✅     |                                                             ❌ (automatic upgrade)                                                              | ❓ [56860](https://shannon.testnet.pokt.network/poktroll/tx/4E201E5C397AB881F417266154C907D38404BE00BE9A443DE28E44A2B09C5CFB) |
```
Why is there a :question: in the "height" column for this row?

okdas replied:
Good call!

bryanchriswhite reviewed (approved):
:ok_hand: Very nice! :sunglasses:

okdas commented:
I've added Beta TestNet table to the list and changed the link to the new Alpha TestNet explorer. No other major changes. Will merge as this was previously approved.
